README.md

```diff
@@ -174,6 +174,112 @@ idxContext = await client.InteractAsync();
+                                            .FirstOrDefault(x => x.Name == "authenticator")
+                                            .Options
+                                            .FirstOrDefault(x => x.Label == "Email")
+                                            .GetProperty<FormValue>("value")
+                                            .Form
+                                            .GetArrayProperty<FormValue>("value")
+                                            .FirstOrDefault(x => x.Name == "id")
+                                            .GetProperty<string>("value");
+
+// Send code
+var selectEmailAuthenticatorRequest = new IdxRequestPayload();
+selectEmailAuthenticatorRequest.StateHandle = identifyResponse.StateHandle;
+selectEmailAuthenticatorRequest.SetProperty("authenticator", new
+{
+    id = emailId,
+});
+
+
+var selectEmailAuthenticatorResponse = await selectEmailAuthenticatorRemediationOption.ProceedAsync(selectEmailAuthenticatorRequest);
+
+var challengeEmailRequest = new IdxRequestPayload();
+challengeEmailRequest.StateHandle = selectEmailAuthenticatorResponse.StateHandle;
+challengeEmailRequest.SetProperty("credentials", new
+{
+    passcode = "xxxxxx",
+});
+
+
+var challengeEmailResponse = await selectEmailAuthenticatorResponse.Remediation.RemediationOptions
+                                                    .FirstOrDefault(x => x.Name == "challenge-authenticator")
+                                                    .ProceedAsync(challengeEmailRequest);
+
+// Send your security question value
+var challengeSecurityQuestionRemediationOption = challengeEmailResponse.Remediation.RemediationOptions
+                                                    .FirstOrDefault(x => x.Name == "challenge-authenticator");
+
+var securityQuestionValue = challengeSecurityQuestionRemediationOption.GetArrayProperty<FormValue>("value")
+                                            .FirstOrDefault(x => x.Name == "credentials")
+                                            .Form.GetArrayProperty<FormValue>("value")
+                                            .FirstOrDefault(x => x.Name == "questionKey")
+                                            .GetProperty<string>("value");
+
+
+var challengeSecurityQuestionRequest = new IdxRequestPayload();
+challengeSecurityQuestionRequest.StateHandle = challengeEmailResponse.StateHandle;
+challengeSecurityQuestionRequest.SetProperty("credentials", new
+{
+    answer = "chicken",
+    questionKey = securityQuestionValue,
+});
+
+var challengeSecurityQuestionResponse = await challengeSecurityQuestionRemediationOption.ProceedAsync(challengeSecurityQuestionRequest);
+
+
+// Reset password
+var resetAuthenticatorRequest = new IdxRequestPayload();
+resetAuthenticatorRequest.StateHandle = challengeSecurityQuestionResponse.StateHandle;
+resetAuthenticatorRequest.SetProperty("credentials", new
+{
+    passcode = "1234Abcd",
```

Comment:
nit: previous passcode was set to `xxxxxx`. For consistency, can we set this to `yyyyyy'?